server/views/pages/prisonerList/index.njk

```diff
@@ -9,6 +9,11 @@
       <li class="govuk-breadcrumbs__list-item">
         <a class="govuk-breadcrumbs__link" href="{{ dpsUrl }}">Digital Prison Services</a>
       </li>
+      {% if userHasPermissionTo('VIEW_SESSION_SUMMARIES') %}
```

Comment:
This is the core fix - if the user has permission to VIEW_SESSION_SUMMARIES they must be a manager with the new Session Summary landing page, and therefore we provide the breadcrumb link back to it